src/main/java/bfk/brickbreaker/BBController.java

```diff
@@ -22,76 +22,77 @@ public BBController(Ball ball, Paddle paddle, BBComponent view, Brick[] bricks)
         this.bricks = bricks;
     }
 
+    public void gamePlay() {
+
+        //ball motion and ball changes direction
+        timer = new Timer(1000 / 60, e -> {
+            System.out.println(ball.getX());
+            System.out.println(ball.x);
+            System.out.println(ball.getY());
+            System.out.println(ball.y);
+            double newX = ball.updateX();
+            double newY = ball.updateY();
+
+            ball.setFrame(newX, newY, ball.width, ball.height);
+            checkCollisions();
+
+
+            view.repaint();
```

Comment:
Put this all in its own method and call it in the timer. 

That way your test can call that method without starting a timer.